pig-runtime/src/org/partiql/pig/runtime/DomainVisitorTransformBase.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package org.partiql.pig.runtime
+
+import com.amazon.ionelement.api.IonElement
+import com.amazon.ionelement.api.MetaContainer
+
+/**
+ * Provides basic a overridable for metas, long and symbol primitives.
```

Comment:
Is this supposed to read "basic and overridable..." I am not sure what basic covers here.

Reply:
Indeed, that is pretty vague.  Will improve this in next commit.